### DISCUSSION TITLE
`fastn migrate` subcommand

### DISCUSSION THREAD

**Discussion by siddhantk232** [fastn-stack/fastn#1638] [Ideas & RFCs]
A wrapper around diesel's migration feature for end users of fastn.

### References

- https://github.com/fastn-stack/fastn/pull/1612#discussion_r1430226147
- https://github.com/orgs/fastn-stack/discussions/1532#discussioncomment-7717866

**Comment by amitu** [comment]
It should be called `fastn migrate`.
